Benchmark build failure (Critereon.Types does not export Pure)

```
Preprocessing library cassava-conduit-0.3.2...
Preprocessing benchmark 'benchmarks' for cassava-conduit-0.3.2...
[1 of 1] Compiling Main             ( benchmarks/Main.hs, dist/build/benchmarks/benchmarks-tmp/Main.o )

benchmarks/Main.hs:10:26: error:
    Module ‘Criterion.Types’ does not export ‘Pure’
```

Here's one way to reproduce:

```
$ stack unpack cassava-conduit-0.3.2 && cd cassava-conduit-0.3.2
$ stack init --resolver nightly-2016-07-19
$ stack bench
```

--- a/benchmarks/Main.hs
+++ b/benchmarks/Main.hs
@@ -7,7 +7,6 @@ import Criterion.Main
     ,   nf
     ,   whnf
     )
-import Criterion.Types ( Pure )
 
 -- | An example function to benchmark...
 --

